src/appleseed.python/bindblenderprogressivetilecallback.cpp

```diff
@@ -172,20 +174,106 @@ namespace
                     glBindTexture(GL_TEXTURE_2D, 0);
                     m_updated_buffer = false;
                 }
-
-                glColor4f(1.0f, 1.0f, 1.0f, 1.0f);
-
-                glEnable(GL_TEXTURE_2D);
-                glBindTexture(GL_TEXTURE_2D, m_texture_id);
-                glBegin(GL_QUADS);
-                    glTexCoord2f(0.0f, 1.0f); glVertex3f(x    ,  y    , 0.0f);
-                    glTexCoord2f(1.0f, 1.0f); glVertex3f(x + w,  y    , 0.0f);
-                    glTexCoord2f(1.0f, 0.0f); glVertex3f(x + w,  y + h, 0.0f);
-                    glTexCoord2f(0.0f, 0.0f); glVertex3f(x    ,  y + h, 0.0f);
-                glEnd();
-                glBindTexture(GL_TEXTURE_2D, 0);
-                glDisable(GL_TEXTURE_2D);
-            }
+                if(!m_updated_data_buffer)
```

Comment:
Please follow our coding conventions detailed here: https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines

That means: 4 spaces for indentation, insert space after `if`, `for`, `while`, etc.

Reply:
Not resolved :)